experimental/manual/oss_fuzz_vuln_prompt.py

```diff
@@ -56,15 +57,17 @@
 * Create a patch that completely addresses the root cause of the vulnerability.
 * Return the patch in git diff format (Use prefix '+' for added and '-' for removed lines).
 * Ensure the patch applies cleanly on the source code files provided.
-* Ensure the patch preserves the intended functionality of the codebase without introducing regressions.
-* Thoroughly test the patch in the relevant environment to verify that it eliminates the crash and prevents exploitation.
+* Ensure the patch preserves the intended functionality of the codebase without introducing functional and performance regressions.
+* Thoroughly analyze the patch to verify that it does not introduce a new security vulnerability.
+* Thoroughly analyze the patch to verify that it eliminates the crash and prevents exploitation.
 """
 PROMPT_MAX_LENGTH = 1048576
 PROJECTS_DIR = os.path.join('oss-fuzz', 'projects')
 STACK_FRAME_START_REGEX = re.compile(r'\s*#\d+\s+0x[0-9A-Fa-f]+\s+')
 STACK_FRAME_PATH_LINE_REGEX = re.compile(
     r'(?<=\[|\(|\s)([a-zA-Z/.][^\s]*?)\s*(:|@)\s*(\d+)(?=\]$|\)$|:\d+$|$)')
-EXCLUDED_FILE_PATH_SUBSTRINGS = ('/compiler-rt/', '/glibc-')
+EXCLUDED_FILE_PATH_SUBSTRINGS = ('/compiler-rt/', '/glibc-',
+                                 '/usr/local/include/')
```

Comment:
Should we just exclude everything under `/usr/` since they're unlikely to contain project or user code?

Reply:
Yeah, let me exclude it in #483